src/transformers/configuration_t5.py

```diff
@@ -57,6 +57,8 @@ class T5Config(PretrainedConfig):
             Size of the intermediate feed forward layer in each :obj:`T5Block`.
         num_layers (:obj:`int`, `optional`, defaults to 6):
             Number of hidden layers in the Transformer encoder.
+        decoder_layers (:obj:`int`, `optional`, defaults to num_layers):
```

Comment:
Documented defaults should match the signature. It will eventually default to `num_layers` and this should be said, but in the line after, suggestion:
```
        decoder_layers (:obj:`int`, `optional`):
            Number of hidden layers in the Transformer decoder. Will use the same value as :obj:`num_layers` if not set.
```
```